CHANGELOG_ANDROID.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## [1.31.87](https://github.com/brave/brave-browser/releases/tag/v1.31.87)
+
+ - Added support for custom filter lists in shields via brave://adblock. ([#8107](https://github.com/brave/brave-browser/issues/8107))
+ - Added support to retrieve New Tab Page background images via CRX which reduces the size of the initial app. ([#17328](https://github.com/brave/brave-browser/issues/17328))
+ - Added ability to disable Chrome Sharing Hub under "Appearance" settings. ([#17548](https://github.com/brave/brave-browser/issues/17548))
```

Comment:
Probably should mention as "Added ability to disable Chrome Sharing Hub in share sheet via "Appearance settings"